### PR TITLE
email export: accordion component

### PR DESCRIPTION
**Pull request by martmalo**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please tick the relevant option.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: packages/plugins/accordion/src/utils/index.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { ReactElement } from 'react';
```
Comment (Darginec05) [inline packages/plugins/accordion/src/utils/index.ts:1]: should be removed after adding svg as string

File: packages/plugins/accordion/src/icons/down.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg  viewBox="0 0 30 30"><polygon points="15,17.4 4.8,7 2,9.8 15,23 28,9.8 25.2,7 "></polygon></svg>
```
Comment (Darginec05) [inline packages/plugins/accordion/src/icons/down.svg:1]: should be removed after adding svg as string

File: packages/plugins/accordion/src/utils/index.ts (hunk 1 above)
Comment (martmalo) [reply packages/plugins/accordion/src/utils/index.ts]: fixed in latest commit

File: packages/plugins/accordion/src/plugin/index.tsx [hunk 3]
```diff
@@ -171,6 +173,42 @@ const Accordion = new YooptaPlugin<AccordionElementMap>({
...
+                .map((listItem) =>
+                  listItem.children
+                    .map((item) => {
+                      if (item.type === 'accordion-list-item-heading') {
+                        return `
+                          <tr>
+                            <td style="padding: 1rem 2rem 1rem 0rem; font-weight: bold; position: relative;">
+                              ${item.children.map((child) => child.text).join('')}
+                              <span style="position: absolute; right: 1rem; top: 50%; transform: translateY(-50%);">
+                                ${renderToHtmlString(<DownIcon style={{ width: '12px', height: '100%' }} />)}
```
[the remark this answers is not in the document]
Comment (martmalo) [reply packages/plugins/accordion/src/plugin/index.tsx]: fixed in latest commit

**Review by Darginec05 — APPROVED** [review]

**Pull request merged**